spec.md

```diff
@@ -1470,7 +1470,7 @@ Platforms SHOULD initiate orphan mitigation in the following scenarios:
 | 201 with malformed response | Failure | Yes |
 | All other 2xx | Failure | Yes |
 | 408 | Timeout failure | Yes |
-| All other 4xx | Service Broker rejected request | No |
+| All other 4xx | Request rejected | No |
```

Comment:
I made this change just so the checker doesn't need to look for "5 word" columns to ignore, that's getting kind of long. Plus saying "Service Broker" isn't really necessary, its clear who is doing the rejection.